### PR TITLE
Sentiment display

### LINKED ISSUE #12: Display sentiment analysis
As a user
I want to have a visual representation of the sentiment analysis for this article in the form of a bubble graph
So that it is easy to have a general impression of the sentiment of an article in a fast and convenient way.

User acceptance criteria:
- [ ] Perform sentiment analysis on an article
- [ ] A graph that shows negative and positive sentiments

### PR DESCRIPTION
**Pull request by sophieeejw**
Now when 'visualization' button clicked, it shows bubbles properly, we just need to style it a little bit.

Closes #12

### CODE REVIEW

File: public/js/visualize.js [hunk 1]
```diff
@@ -1,75 +1,105 @@
-var urlParams = new URLSearchParams(window.location.search);
-var url = "visualize/" + urlParams.getAll("articleID");
+const urlParams = new URLSearchParams(window.location.search);
+const url = `visualize/${urlParams.getAll('articleID')}`;
 
-fetchGET(url, function(err, data) {
+fetchGET(url, (err, data) => {
```
Comment (claireinez) [inline public/js/visualize.js]: This is front end code but I don't think you're transpiling back to ES5 in this project. Have you tested this in browsers that aren't so modern to have ES6 built in?

File: src/views/visualize.hbs [hunk 2]
```diff
@@ -1,7 +1,25 @@
...
+      <meta name="viewport" content="width=device-width,
+      initial-scale=1.0">
+      <meta http-equiv="X-UA-Compatible" content="ie=edge">
+      <link rel="stylesheet" href="https://unpkg.com/tachyons@4.9.0/css/tachyons.min.css"/>
+      <link
+      href="https://fonts.googleapis.com/css?family=Roboto"
+      rel="stylesheet">
+      <style type="text/css" media="screen">
+        svg{
+          display: block;
```
Comment (claireinez) [inline src/views/visualize.hbs]: Is this the right place to put this?

File: public/js/visualize.js (hunk 1 above)
Comment (sophieeejw) [reply public/js/visualize.js]: We wrote it ES5 for frontend codes but prettier changed it to ES6 automatically which I didn't realize when I made this PR. Now all the public files are prettier-ignored!

File: src/views/visualize.hbs (hunk 2 above)
Comment (sophieeejw) [reply src/views/visualize.hbs]: this block was moved to CSS file in next PR, so for now I just deleted it.

File: public/js/visualize.js [hunk 3]
```diff
@@ -1,75 +1,105 @@
...
-      .attr("fill", function(d) {
-        return d.color;
-      });
+      .append('g')
+      .call(drag);
 
-    nodes_circles = d3.selectAll("circle");
+    group
+      .append('circle')
+      .attr('cx', d => width / 2)
```
Comment (claireinez) [inline public/js/visualize.js]: These are still arrow functions ;)

Comment (sophieeejw) [reply public/js/visualize.js]: ooooooooh right d3 😖

**Comment by sophieeejw** [comment]
@claireinez thanks for your patience 🙏 we tried our best to go through and change every arrow function 😂

**Pull request merged**